nav2_bringup/params/nav2_params.yaml

```diff
@@ -85,6 +85,7 @@ bt_navigator:
     - nav2_round_robin_node_bt_node
     - nav2_transform_available_condition_bt_node
     - nav2_time_expired_condition_bt_node
+    - nav2_path_expiring_timer_condition
```

Comment:
I know its annoying, but there are other YAML files this should be added to in order to make sure they're loaded for all appropriate tests if we use this in the future.

What I would like to suggest is that you actually **delete** this parameter and list from all non `nav2_bringup`-located yaml files. That will stop us from needing to make this update to 20 files every time we add a new node. That way we only need to update the 3 yamls in Nav2 and the default in BT navigator for them all to have access to all of the given nodes